docs/features/writing-different-telemetry-types.md

```diff
@@ -60,7 +60,7 @@ durationMeasurement.Start();
 var startTime = DateTimeOffset.UtcNow;
 
 _logger.LogBlobStorageDependency(accountName: "multimedia", containerName: "images", isSuccessful: true, startTime, durationMeasurement.Elapsed);
-// Output: "Dependency Azure blob multimedia named images in 00:00:00.2521801 at 03/23/2020 09:56:31 +00:00 (Successful: True - Context: )"
+// Output: "Dependency Azure blob images named multimedia in 00:00:00.2521801 at 03/23/2020 09:56:31 +00:00 (Successful: True - Context: )"
```

Comment:
Apperently, it was only the docs that weren't in sync with the source. The source already got the account name as target name.